test/types/query-types/boolean.test-d.ts

```diff
@@ -47,3 +49,5 @@ expectAssignable<SubsetFilters<{ testField: EntryFields.Boolean }, 'fields'>>({
   'fields.testField[in]': booleanValue,
   'fields.testField[nin]': booleanValue,
 })
+
+// expectType<Required<FullTextSearchFilters<{ testField: EntryFields.Boolean }, 'fields'>>>({})
```

Comment:
Why is this commented out? Is it a todo?

Reply:
that was me testing sth out, thank you for noticing it!